andela/travel_tool_front
# #160370617 One way trip with new request modal

#### What does this PR do?
- Update the new request modal based on the invisionapp prototype and allow the user to choose the type of trip they want from the list of trip types

#### Description of Task to be completed?
- Update the new request modal based on the invisionapp prototype 
- Allow the user to choose the type of trip they want from the list of trip types


#### How should this be manually tested?
- Clone this repo - `git clone https://github.com/andela/travel_tool_front.git`
- Switch to the repo directory - `cd travel_tool_front`
- Check out this branch - `ft-one-way-trip-modal-160370617`
- Install dependencies and start the server  - `make start`
- Set up the server by following this [guide](https://github.com/andela/travel_tool_back/pull/44)
- Navigate to `localhost:3000` in a browser
- Click on create a new request and select the type of request from the options available 

#### Any background context you want to provide?
- You need a google maps API key to use the autocomplete feature
- Add the CDN with the API key to the dot env with the default key of `REACT_APP_CITY`=`https://maps.googleapis.com/maps/api/js?key=YOUR-API-KEY&libraries=places`
- Validation for input field does not work as expected 
- [invisionapp](https://projects.invisionapp.com/d/main#/console/14861108/319652028/preview)


#### What are the relevant pivotal tracker stories?
[160370617](https://www.pivotaltracker.com/story/show/160370617)

#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/20597418/45558386-b558ae00-b837-11e8-8337-2ef88448c13d.png)

![image](https://user-images.githubusercontent.com/20597418/45558348-9eb25700-b837-11e8-939d-e0da8f17a243.png)




#### Questions:
None
